src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -255,10 +258,6 @@ private void validateLadderOrientation()
     {
         BuildingMiner buildingMiner = getOwnBuilding();
 
-        int x = buildingMiner.getLadderLocation().getX();
-        int y = buildingMiner.getLadderLocation().getY();
-        int z = buildingMiner.getLadderLocation().getZ();
-
         //TODO: for 1.8 change to getBlockState
         EnumFacing ladderOrientation = world.getBlockState(buildingMiner.getLadderLocation()).getValue(BlockLadder.FACING);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'ladderOrientation' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)